native/cocos/editor-support/spine-creator-support/SpineAnimationState.h

```diff
@@ -0,0 +1,53 @@
+/******************************************************************************
+ * Spine Runtimes License Agreement
+ * Last updated January 1, 2020. Replaces all prior versions.
+ *
+ * Copyright (c) 2013-2020, Esoteric Software LLC
```

Comment:
This is a new file we created, it's not a file in spine runtime , so I think we need to use our license.